src/analysis/analysis.jl

```diff
@@ -1,10 +1,11 @@
 module analysis
 
-using Statistics, DataFrames
-using NamedDims, AxisKeys
 using ADRIA: ResultSet, n_locations
 using ADRIA.metrics: nds
 
+using JuliennedArrays: Slices
+using NamedDims, AxisKeys
+using Statistics, DataFrames
```

Comment:
😮 but this is not Blue style!

Kidding - fine to leave as is for now. Let's clean up later.